Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputManager.h

```diff
@@ -54,8 +65,12 @@ namespace AzToolsFramework
         //! Internally, this will reset the cursor position after each move event to ensure movement
         //! events don't allow the cursor to escape. This can be used for typical camera controls
         //! like a dolly or rotation, where mouse movement is important but cursor location is not.
+        //! @deprecated Use #SetCursorMode()
```

Comment:
+1 on leaving the deprecated API in place, one thing to try and do is record this deprecation and know when to update/remove it in future. It may be worth making a GHI to track this (see this PR for more info on proposed deprecation process - https://github.com/o3de/sig-release/pull/23)